### PR TITLE
Adding syntax of ternary operator in c.md

### PR DESCRIPTION
**Pull request by sahiljawale54**
I am adding the syntax of the ternary operator in the c.md file 

I hope u find this little contribution worth merging 
Thank You

### CODE REVIEW

File: c.md [hunk 1]
```diff
@@ -238,6 +238,12 @@ if(conditional-expression)
     //code
 }
 ```
+### 2.1. If-else using Ternary Operator
+```c
+conditional-expression ? code1 : code2;
+```
+If conditional-expression is true, code1 is executed. And, if condition is false, code2 is executed.
```
Comment (onecompiler-ops) [inline c.md:245]: Instead this, can you add self explanatory code block? like below 

`bool isEven = (i % 2 == 0) ? true : false;`

**Review by onecompiler-ops — CHANGES_REQUESTED** [review]

File: c.md (hunk 1 above)
Comment (sahiljawale54) [reply c.md:245]: Yes, surely
can I go with the same example as mentioned?
@onecompiler-ops

Comment (onecompiler-ops) [reply c.md:245]: Yes, please test once and add it

**Review by Sahiljawale — APPROVED** [review]

**Review by onecompiler-ops — COMMENTED** [review]
//isEven = 0 

this should be 

//isEven = true

**Comment by sahiljawale54** [comment]
Actually, there is no true/false in c language it uses 1/0 only 

bool isEven = (4 % 2 == 0) ? true: false;

If we run this line of code and the output value of 'isEven' then we will get 1 as the output

**Comment by sahiljawale54** [comment]
image

I have added screenshot here

**Pull request closed without merging**